test/models/album_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class AlbumTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
```

Comment:
I believe I was working on the tests as you were reviewing this code. They will go up in a push later tonight.